samples/hello_world/prj.conf

```diff
@@ -1 +1,2 @@
 # nothing here
+CONFIG_MULTITHREADING=n
```

Comment:
What was the reason for this?

Reply:
Yeah, that's wrong.  Maybe a debug that slipped in and everybody missed it in the cmake noise?

Reply:
Somebody was fast: #28671

Reply:
Though, actually, it would make some sense for hello-world to be a single-thread example :)